src/main/java/com/softech/coursesvc/controller/CourseController.java

```diff
@@ -38,16 +35,29 @@ public CourseList getCourses() {
     }
 
 @GetMapping("/v1/course/{courseId}")
-public ResponseEntity<CourseEntity> getCourseById(@PathVariable Long courseId)
-{
-    CourseEntity course= courseSvc.getCourseById(courseId);
-    if(course != null)
-    {
+public ResponseEntity<CourseEntity> getCourseById(@PathVariable Long courseId) {
+    CourseEntity course = courseSvc.getCourseById(courseId);
```

Comment:
Are you working on get by id endpoint? I guess neetu worked on it..why you have this added

Reply:
Line 38 39, if you added them , remove them

Reply:
She had to work on create, update and delete endpoint

Reply:
When I clone the project, it automatically appears in my project list